#use wml::debian::translation-check translation="1.1" maintainer="Giuseppe Sacco"
<define-tag description>file temporanei insicuri</define-tag>
<define-tag moreinfo>
<p>Zack Weinberg ha scoperto un uso insicuro di file temporanei in
os._execvpe di os.py. Utilizza un nome prevedibile che può portare alla
esecuzione di un codice arbitrario.</p>

<p>Questo problema è stato risolto in varie versioni di Python: Per la
attuale distribuzione stable (woody) è stato risolto nella versione
1.5.2-23.1 di Python 1.5, nella versione 2.1.3-3.1 di Python 2.1 e nella
versione 2.2.1-4.1 di Python 2.2. Per la vecchia distribuzione stable
(potato) questo è stato risolto nella versione 1.5.2-10potato12 per Python
1.5. Per la distribuzione unstable (sid) questo è stato risolto nella
versione 1.5.2-24 di Python 1.5, nella versione 2.1.3-6a di Python 2.1 e
nella versione 2.2.1-8 di Python 2.2. Python 2.3 non è affetto da questo
problema.</p>

<p>Suggeriamo di aggiornare immediatamente i propri pacchetti Python.</p>
</define-tag>

# do not modify the following line
#include "$(ENGLISHDIR)/security/2002/dsa-159.data"
